scripts/create_type_data.sql

```diff
@@ -23,7 +23,10 @@ INSERT INTO bookbrainz.achievement_type ("name", "description", "badge_url") VAL
 	('Limited Edition III', 'Create 100 editions', '/images/revisionistI.svg'),
 	('Worker Bee I', 'Create one work', '/images/revisionistI.svg'),
 	('Worker Bee II', 'Create 10 works', '/images/revisionistI.svg'),
-	('Worker Bee III', 'Create 100 editions', '/images/revisionistI.svg');
+	('Worker Bee III', 'Create 100 editions', '/images/revisionistI.svg'),
+	('Explorer I', 'View 1 different entities', '/images/revisionistI.svg'),
```

Comment:
Hmm, these tiers seem wrong - I think it should be 10, 100, 1000 (see https://chatlogs.metabrainz.org/brainzbot/metabrainz/2016-07-02/?msg=3640137&page=2)